### PR TITLE
[CN-536] Add ReplicatedMap CR documentation

### PR DESCRIPTION
**Pull request by cagric0**

### CODE REVIEW

File: docs/modules/ROOT/pages/replicatedmap-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+|Name of the ReplicatedMap configuration that you're creating. If empty, the custom resource (CR) name is used.
+
+|hazelcastResourceName
+|Defines the name of the Hazelcast resource.
+
+|inMemoryFormat
+|Internal storage format. Use either OBJECT or BINARY.
+
+|asyncFillup
+|Specifies whether the ReplicatedMap is available for reads before the initial replication is completed. Its default value is true. If set to false, i.e., synchronous initial fill up, no exception is thrown when the ReplicatedMap is not yet ready, but null values can be seen until the initial replication is completed.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/replicatedmap-configuration.adoc:24]:
```suggestion
|Specifies whether the ReplicatedMap is available for reads before the initial replication is completed. The default value is true. If set to false, a synchronous initial fill up the ReplicatedMap is allowed, and no exceptions are thrown until the initial replication is complete. Null values may be returned while the initial replication is completed.
```
I think this is accurate but please do check the suggested changes.

**Review by angelasimms — CHANGES_REQUESTED** [review]
Minor text updates requested.

File: docs/modules/ROOT/pages/replicatedmap-configuration.adoc (hunk 1 above)
Comment (cagric0) [reply docs/modules/ROOT/pages/replicatedmap-configuration.adoc:24]: After your comment I checked the replicated map doc page and I think the description for `async-fillup` field is wrong and I created a github issue (https://github.com/hazelcast/hz-docs/issues/568). Could you take a look at it?

Comment (angelasimms) [reply docs/modules/ROOT/pages/replicatedmap-configuration.adoc:24]: Hi there, I've updated the description above. If you're happy that it is accurate, I'll update the replicated map page with the same description. Thanks for spotting this.

Comment (angelasimms) [reply docs/modules/ROOT/pages/replicatedmap-configuration.adoc:24]: Hi @cagric0. I've updated the description above. If this is OK, I'll add the updated description to the Replicated Map page. Thanks again for spotting this.

Comment (angelasimms) [reply docs/modules/ROOT/pages/replicatedmap-configuration.adoc:24]: Yes, the more direct the better. I've updated the description to match this.

**Review by angelasimms — APPROVED** [review]
All looks good now.

**Pull request merged**